public/javascripts/SVLabel/src/SVLabel/Keyboard.js

```diff
@@ -30,13 +30,62 @@ function Keyboard ($) {
         $(document).bind('keyup', documentKeyUp);
         $(document).bind('keydown', documentKeyDown);
     }
+    function toRadians (angle) {
+        return angle * (Math.PI / 180);
+    }
+    // Move in the direction of a link closest to a given angle
+    function movePano(angle) {
+        // take the cosine of the difference for each link to the current heading in radians and stores them to an array
+        var cosines = svl.panorama.links.map(function(link) { return Math.cos(toRadians(svl.panorama.pov.heading + angle) - toRadians(link.heading))});
+        // finds index of greatest value in cosines array
+        var maxIndex = cosines.reduce((iMax, x, i, arr) => x > arr[iMax] ? i : iMax, 0);
```

Comment:
Unfortunately `=>` operator is not supported on Safari and IE :( You have to use an old anonymous function.